maplibre/src/context.rs

```diff
@@ -12,9 +14,12 @@ pub struct ViewState {
 }
 
 impl ViewState {
-    pub fn new(window_size: &WindowSize) -> Self {
+    pub fn new<P: Into<cgmath::Rad<f64>>>(window_size: &WindowSize, fovy: P) -> Self {
+        let center = TILE_SIZE / 2.0;
+        let fovy = fovy.into();
+        let height = center / (fovy / 2.0).tan();
         let camera = Camera::new(
-            (TILE_SIZE / 2.0, TILE_SIZE / 2.0, 150.0),
+            (center, center, height),
```

Comment:
I think we should still get the default values from the style configuration instead of showing the entire map. I can make that separatly in another PR if you agree?

Reply:
The height/fovy is not defined in the style.
The center position gets set in another PR.